edit/edit.go

```diff
@@ -520,6 +520,8 @@ func match(ed *Editor, at addr, re *re1.Regexp) ([][2]int64, error) {
 // Items in {} are optional.
 //
 // Commands are:
+//	addr
+//		addr sets the address of Dot.
```

Comment:
I would've just said "Sets the address of Dot."
The "addr" comes from the word addr on the previous line.